Project/code for db schema.txt

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE User("ID" INT,"Email" TEXT,"Username" TEXT, "Password" TEXT);
+CREATE TABLE GameMap("CoOrdinateID" INT,"CoOrdinateX" INT,"CoOrdinateY" INT, "CoOrdinateZ" INT);
+CREATE TABLE MegaFeatures("FeaturesID" INT,"Posts" TEXT,"Portfolios" TEXT);
+CREATE TABLE Dashboard("DashboardID" INT,"DashboardFeatures" TEXT);
+CREATE TABLE Posts("PostID" INT,"Posts" TEXT);
+CREATE TABLE Portfolio("PortfolioID" INT,"Portfolios" TEXT);
+.schema
```

Comment:
@code93 You were supposed to create a separate PR with schema so that we can analyze it properly. 
Please create `.sql` files for your schema, and create a separate PR. Make sure you run an auto-formatter on your SQL code. `.sql` files can be executed by your db (sqlite in our case) and are real verifiable code; this is just documentation.